pkg/webui/console/views/application/index.js

```diff
@@ -196,6 +197,18 @@ import Devices from '../devices'
 })
 @withEnv
 export default class Application extends React.Component {
+  static propTypes = {
+    appId: PropTypes.string.isRequired,
+    application: PropTypes.application.isRequired,
+    env: PropTypes.env,
```

Comment:
```suggestion
    env: PropTypes.env.isRequired,
```
If I don't miss anything, the env prop can also be just required.